src/space-connector/type.ts

```diff
@@ -2,11 +2,41 @@ export interface SessionTimeoutCallback {
     (): void;
 }
 
+export type ResponseCode = 'APIError' | 'NotFoundError' | 'BadRequestError' | 'AuthenticationError' | 'AuthorizationError'
+
+export interface Response<T> {
+    code: ResponseCode | string;
+    message: string;
+    result: T;
+}
+
 export interface APIInfo {
     path: string;
     methods: Array<string>;
 }
 
+export interface ErrorModel {
+    message: string;
+    code: string;
+}
+
+export interface ServerError {
+    status: string;
+    error: ErrorModel;
+    statusText: string;
+}
+
+export interface AxiosPostResponse {
+    access_token: string;
+    refresh_token: string;
+    apis: APIInfo[];
+}
+
+export interface ServerResponse<T> {
+    results: T
+    total_count: number
+}
```

Comment:
```suggestion
export interface ServerListResponse<T> {
    results: T
    total_count: number
}
```

모든 서버에 대한 응답이 저렇게 오지는 않고,  list api 요청에 대하여서만 저렇게 오는 것으로 알고 있어요!
그게 타입 이름에 녹아져있으면 더 좋을 것 같습니다

Reply:
엇 그러고보니 이렇게 되는 것이 더 좋겠네요.
```suggestion
export interface ServerResponse<T> {
    results: T[];
    total_count: number;
}
```
일반적으로 제너릭으로 받은 타입의 배열을 results 로 올 것 같아요.